dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -522,6 +526,12 @@ def runCrossValidation(self, jobId, interfaces):
         errorDb.writeAllRowErrors(jobId)
         interfaces.jobDb.markJobStatus(jobId, "finished")
         CloudLogger.logError("VALIDATOR_INFO: ", "Completed runCrossValidation on submissionID: "+str(submissionId), "")
+        # Update error info for submission
+        interfaces.jobDb.getSubmissionErrorInfo(submissionId)
+        # TODO: Remove temporary step below
+        # Temporarily set publishable flag at end of cross file, remove this once users are able to mark their submissions
+        # as publishable
+        interfaces.jobDb.setPublishableFlag(submissionId, True)
```

Comment:
The purpose of this flag is to represent whether the data store should actually publish the submission.  It is supposed to be set when the user clicks publish on the frontend (which they can only do after validating without errors).  Setting it here is a temporary workaround to let the data store have submissions be marked as ready to publish before the functionality is added on the frontend.  I could add a check on number of errors here so that only submissions without errors are marked as publishable, since that's a better reflection of the final process.